wrappers/java/src/test/java/org/hyperledger/indy/sdk/wallet/ImportWalletTest.java

```diff
@@ -0,0 +1,44 @@
+package org.hyperledger.indy.sdk.wallet;
+
+import org.hyperledger.indy.sdk.IOException;
+import org.hyperledger.indy.sdk.IndyIntegrationTestWithSingleWallet;
+import org.hyperledger.indy.sdk.did.Did;
+import org.junit.Test;
+
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.junit.Assert.assertEquals;
+
+
+public class ImportWalletTest extends IndyIntegrationTestWithSingleWallet {
```

Comment:
this is actually export-import test, so the other (export) test class is redundant

Reply:
@nveskovic We use one module per each API endpoint. It can cause some code duplication, but it is a warranty of formal proof for each endpoint.